### PR TITLE
test bluechictl status -w

### LINKED ISSUE #785: Add integration test for bluechictl status -w
Add integration test(s) to verify `bluechictl status -w` functionality.

### PR DESCRIPTION
**Pull request by mkemel**
Add a test that covers the functions handling the `bluechictl status -w` command

Fixes: https://github.com/eclipse-bluechi/bluechi/issues/785

### CODE REVIEW

File: tests/tests/tier0/bluechictl-status-watch/python/bluechictl_status_watch.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    def setUp(self) -> None:
+        node_status_offline = re.compile(r"^" + re.escape(node_foo_name) + ".*offline")
+        node_status_online = re.compile(r"^" + re.escape(node_foo_name) + ".*online")
+        expected_patterns = {node_status_online,
+                             node_status_offline}
+        args = ["status", "-w"]
+        self.evaluator = SimpleEventEvaluator(expected_patterns)
+        self.bgrunner = BackgroundRunner(args, self.evaluator)
+        self.bluechictl = BluechiCtl()
```
Comment (engelmi) [inline tests/tests/tier0/bluechictl-status-watch/python/bluechictl_status_watch.py]: `self.bluechictl` isn't used and can be removed.

File: tests/tests/tier0/bluechictl-status-watch/main.fmf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+summary: Test bluechictl status functionality
```
Comment (engelmi) [inline tests/tests/tier0/bluechictl-status-watch/main.fmf]: NIT:
```suggestion
summary: Test bluechictl status watch functionality
```

File: .clang-tidy [hunk 3]
```diff
@@ -24,7 +24,8 @@ Checks: >
     -readability-function-cognitive-complexity,
     -modernize-macro-to-enum,
     -llvm-else-after-return,
-    -readability-else-after-return
+    -readability-else-after-return,
+    -misc-include-cleaner
```
Comment (engelmi) [inline .clang-tidy:28]: Just FYI (feel free to resolve anytime):
We pushed a new build-base image yesterday and it seems a new clang-tidy version with this new rule was installed there. All detected "errors" are unrelated to this PR, so its fine to disable it for now.

Comment (mkemel) [reply .clang-tidy:28]: Yes, I figured that this is the case.
Anyway, I don't think we should enforce it.

File: tests/tests/tier0/bluechictl-status-watch/python/bluechictl_status_watch.py (hunk 1 above)
Comment (mkemel) [reply tests/tests/tier0/bluechictl-status-watch/python/bluechictl_status_watch.py]: Done, thanks

File: .clang-tidy (hunk 3 above)
Comment (engelmi) [reply .clang-tidy:28]: Agreed.

**Review by engelmi — APPROVED** [review]
LGTM

**Pull request merged**